StorageUtility/Windows/DeviceHandle.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+<License>
+Copyright 2015 Virtium Technology
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http ://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+</License>
+*/
+#include "Device.h"
+#include "DeviceHandle.h"
+#include "StorageUtility.h"
+
+namespace vtStor
+{
+
+cDeviceHandle::cDeviceHandle(std::shared_ptr<cDeviceInterface> Device)
+{
+    String devicePath;
+    GetDevicePath(devicePath, Device);
+    GetStorageDeviceHandle(devicePath, m_DeviceHandle);
+    //TODO: handle errors
+}
+
+cDeviceHandle::~cDeviceHandle()
+{
+    CloseDeviceHandle(m_DeviceHandle);
```

Comment:
Nothing should be done here. It is the user responsibility to close the device handle.